app/components/CommitNavbar.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import { Nav, Navbar } from "react-bootstrap";
+import * as styles from "./CommitNavbar.scss";
+
+// it is currently assumed that Cancel component will change the state and clear everything
+export const CancelCommit: React.SFC = () => <a href="#" className={styles.clear} />;
+
+export const CommitNavbar: React.SFC = () =>
```

Comment:
This component should be able to receive and display children (title etc).  This should be tested as well
![image](https://user-images.githubusercontent.com/1814312/30030852-54fe53e0-918f-11e7-9468-d4af2abea837.png)